### PR TITLE
Update RAT-Partner page: Add information about outdated page

### LINKED ISSUE #3466: Obsolete rat-partner
https://www.coronawarn.app/en/rat-partner/ is obsolete and should be removed, or at least the text replaced.

image

It is still listed in the [sitemap](https://www.coronawarn.app/en/sitemap/)

image

and it can be found by Google search.

Internal Tracking ID: [EXPOSUREAPP-15081](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-15081)

### PR DESCRIPTION
**Pull request by larswmh**
This PR closes #3466 

EN:
image

DE:
image

---
Internal Tracking ID: [EXPOSUREAPP-15081](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-15081)

### CODE REVIEW

**Review by dsarkar — APPROVED** [review]

**Comment by MikeMcC399** [comment]
For both German and English I would change the order and put the fact that the information is no longer relevant at the beginning of the text.

The other details in this additional text are really also not relevant anymore, but if there is some reason to add these details they should follow on underneath.

File: src/data/rat-partner.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "section-notice": {
         "textblock": [
+            "<div class='update'>Due to the test regulation of 31.03.2022, test sites are no longer obliged to connect to the Corona-Warn-App of the Robert Koch-Institute, therefore registrations for API connection and connection via the CWA rapid test portal are no longer possible. Interested parties who signed a contract before 01.04.2022 were still connected until 13.05.2022. The test infrastructure was deactivated on April 19, 2023 anyway. Thus, the information on the page is no longer up to date.</div>",
```
[the remark this answers is not in the document]
Comment (MikeMcC399) [reply src/data/rat-partner.json:4]: Suggest as introduction / heading:

The test infrastructure was deactivated on April 19, 2023 and the information on this page is no longer up to date.

File: src/data/rat-partner_de.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
     "section-notice": {
         "textblock": [
+            "<div class='update'>Aufgrund der Testverordnung vom 31.03.2022 müssen sich Teststellen nicht mehr verpflichtend an die Corona-Warn-App des Robert Koch-Instituts anbinden, weshalb Anmeldungen zur API-Anbindung und Anbindung über das CWA Schnelltestportal nicht mehr möglich sind. Interessenten, die vor dem 01.04.2022 einen Vertrag abgeschlossen haben, wurden bis zum 13.05.2022 noch fertig angebunden. Ohnehin wurde die Test-Infrastruktur am 19. April 2023 abgeschaltet. Damit sind die Informationen auf der Seite nicht mehr aktuell.</div>",
```
Comment (MikeMcC399) [inline src/data/rat-partner_de.json]: Begin with:

Die Test-Infrastruktur wurde am 19. April 2023 abgeschaltet. Damit sind die Informationen auf der Seite nicht mehr aktuell.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for your review and comments!

File: src/data/rat-partner.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
     "section-notice": {
         "textblock": [
+            "<div class='update'>The information on the page is no longer up to date.<br />Due to the test regulation of March 31, 2022, test sites are no longer obliged to connect to the Corona-Warn-App of the Robert Koch-Institute, therefore registrations for API connection and connection via the CWA rapid test portal are no longer possible. Interested parties who signed a contract before April 1, 2022 were still connected until May 13, 2022. The test infrastructure was deactivated on April 19, 2023 anyway.</div>",
             "Dear interested persons,",
             "Thank you for your interest in connecting to the Corona-Warn-App. You can find current information about this on <a href='https://github.com/corona-warn-app/cwa-quicktest-onboarding/wiki' target='_blank' rel='noopener noreferrer'>GitHub</a>.",
             "Sincerely",
```
Comment (Ein-Tim) [inline src/data/rat-partner.json:7]: Do we still need this after adding the other update?

**Pull request merged**